handling-incidents.md

```diff
@@ -0,0 +1,101 @@
+## Handling Standards of Conduct Incidents
+
+This document is for event organizers and volunteers and provides guidelines
+on handling violations of the standards of conduct as well as incident
+complaints from attendees.
+
+## SoC Event Committee
+
+All TPF events will have a per-event Standards of Conduct committee appointed
+to act as the final authority for all issues related to the standards of
```

Comment:
s/final authority/delegated authority/       (Or perhaps "designated"?)    

Ultimately at the end of the day, the people that are going to be held responsible if something goes wrong are the people that have the authority.    That will include the lead organizers and TPF's Board of Directors.    But the SoC is empowered and authorized to handle incidents with the expectation that the Board will seldom, if ever, need to get involved.